wbooks-api/app/views/layouts/mailer.html.erb

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
```

Comment:
Another comment that's not a correction: when starting a pure API project try to remove stuff that you won't use like views, js and css.